custom/dhis2/tasks.py

```diff
@@ -288,64 +214,60 @@ def gen_children_only_ours(domain):
             yield CommCareCase.wrap(doc)
 
 
-@periodic_task(run_every=crontab(minute=4, hour=4))  # Run daily at 04h04
-def sync_child_entities():
+# TODO: Use case forwarding, or form forwarding of registration forms
+@periodic_task(run_every=timedelta(minutes=5))  # Run every 5 minutes to match forwarded forms
+def sync_cases():
```

Comment:
if the DHIS configs live in separate documents one simple thing would be to just move them to the meta database and make a view.